server/routes/api/v1/admin/surveys.js

```diff
@@ -187,60 +187,41 @@ router.get('/csv', auth.ensureLoggedIn, auth.ensureAdmin, function(req, res) {
       });
     });
   } else if (include === 'last' || include === 'first') {
+      var order = include
+    var surveyCountParam = ids.length - 1;
+
+    var q = knex.select(knex.raw('subquery.*, a.value, q.text'))
```

Comment:
knex isn't defined.  It is referenced elsewhere with bookshelf.knex.  You could just say var knex = bookshelf.knex